src/main/java/buildcraftAdditions/blocks/BlockTest.java

```diff
@@ -17,10 +18,13 @@
  * Please check the contents of the license located in
  * http://buildcraftadditions.wordpress.com/wiki/licensing-stuff/
  */
-public class PaternTestingBlock extends Block {
+public class BlockTest extends Block {
 
-	public PaternTestingBlock() {
+	public BlockTest() {
 		super(Material.iron);
+		setBlockName("testBlock");
+		setBlockTextureName("bcadditions:testBlock");
```

Comment:
Just wanted to get rid of those Exceptions in the console to filter out Execptions which should be fixed